### PR TITLE
Fixing some unused-result warnings

### PR DESCRIPTION
**Pull request by smalls12**
Pertains to the `GuiLoadStyle()` function which reads from files.
`fgets()` and `fread()` error codes are not checked.
If any of these commands fail, I am not sure what to do at that point.
As it is right now, it just outright fails and will close the file.
This pull request is mainly to highlight the issue, I can make any changes requested to align with the style guide.

### CODE REVIEW

File: src/raygui.h [hunk 1]
```diff
@@ -2896,68 +2896,72 @@ RAYGUIDEF void GuiLoadStyle(const char *fileName)
     if (rgsFile != NULL)
     {
         char buffer[256] = { 0 };
-        fgets(buffer, 256, rgsFile);
-
-        if (buffer[0] == '#')
+        if ( fgets(buffer, 256, rgsFile) != NULL )
```
Comment (smalls12) [inline src/raygui.h]: fgets will return nullptr on error.
[https://www.tutorialspoint.com/c_standard_library/c_function_fgets.htm](url)

Comment (smalls12) [reply src/raygui.h]: This looks like I did more changes than required, just tabbed into the if statement.

File: src/raygui.h [hunk 2]
```diff
@@ -2896,68 +2896,72 @@ RAYGUIDEF void GuiLoadStyle(const char *fileName)
...
-                    default: break;
-                }
+                        } break;
+                        default: break;
+                    }
 
-                fgets(buffer, 256, rgsFile);
+                    if ( fgets(buffer, 256, rgsFile) == NULL )
+                    {
+                        // error
```
Comment (smalls12) [inline src/raygui.h]: I wasn't sure what to do here, how do we propagate errors to the user in this case?

File: src/raygui.h [hunk 3]
```diff
@@ -2973,10 +2977,15 @@ RAYGUIDEF void GuiLoadStyle(const char *fileName)
         short reserved = 0;
         int propertiesCount = 0;
 
-        fread(signature, 1, 4, rgsFile);
-        fread(&version, 1, sizeof(short), rgsFile);
-        fread(&reserved, 1, sizeof(short), rgsFile);
-        fread(&propertiesCount, 1, sizeof(int), rgsFile);
+        int rc = 0;
+        rc = fread(signature, 1, 4, rgsFile);
```
Comment (smalls12) [inline src/raygui.h]: fread errors
```The total number of elements successfully read are returned as a size_t object, which is an integral data type. If this number differs from the nmemb parameter, then either an error had occurred or the End Of File was reached.```
[https://www.tutorialspoint.com/c_standard_library/c_function_fread.htm](url)

File: src/raygui.h [hunk 4]
```diff
@@ -3067,6 +3097,7 @@ RAYGUIDEF void GuiLoadStyle(const char *fileName)
 #endif
         }
 
+close:
```
Comment (smalls12) [inline src/raygui.h]: On fread errors, the function will jump down to here.

**Comment by smalls12** [comment]
Contributes to #61

**Comment by raysan5** [comment]
@smalls12 Excuse me but I try to avoid `goto` in code. Despite changes are for good I think they make readability more complex, I wont' merge those changes for the moment. Sorry for the inconvenience.

**Comment by smalls12** [comment]
No problem, I made this small pull request to minimize my effort to test the waters.
The apprehension towards goto should be captured in the contribution guidelines, this does not exist currently.
As for readability, there must be some middle ground as the warnings can actually be the cause for some errors.

**Pull request closed without merging**